### DISCUSSION TITLE
Unable to build image with windows native container image(nanoserver) on Rancher Desktop

### DISCUSSION THREAD

**Discussion by Annie-12** [rancher-sandbox/rancher-desktop#1454] [Q&A] [answered]
I am trying to build an image using Dockerfile that uses  `mcr.microsoft.com/dotnet/aspnet:5.0-nanoserver-1809` as base . I have tried with both containerd and dockerd runtimes but am not being able to build and run it using **Rancher Desktop**.

- For containerd , though the` nerdctl build` is working but `nerdctl run` throws the error:
`FATA[0000] mount callback failed on /tmp/containerd-mount3046644623: open /tmp/containerd-mount3046644623/etc/passwd: no such file or directory`
- For dockerd , while docker build it throws :
`image operating system "windows" cannot be used on this platform`

I know there is a way to switch daemon to windows in docker desktop . Is there any way to resolve that issue in rancher desktop?( I need to run the nanoserver image)
OS: WIndows 10 Enterprise

**Comment by mook-as** [comment, marked as answer]
We do not support Windows containers at the moment, only Linux ones (x86_64 exclusively, except on M1 macs where it's arm64 exclusively).  Please refer to https://github.com/rancher-sandbox/rancher-desktop/issues/255 for details.

**Reply by jandubois** [reply]
Irrelevant to the question, but this is not quite correct, we do support both x86_64 and aarch64 Linux containers on either Mac architecture (via `binfmt_misc` and `qemu-static` inside the VM), when using the container runtime directly (not using k8s), e.g.

```console
$ nerdctl run --platform linux/arm64 busybox uname -a
Linux 642f631c0d94 5.10.82-0-virt #1-Alpine SMP Fri, 26 Nov 2021 13:19:46 +0000 aarch64 GNU/Linux

$ nerdctl run --platform linux/amd64 busybox uname -a
Linux bf243802ec6d 5.10.82-0-virt #1-Alpine SMP Fri, 26 Nov 2021 13:19:46 +0000 x86_64 GNU/Linux
```

We do not support `linux/arm64` images on Windows right now, and we don't have support for Windows containers on any platform yet.
